submissions/Dev focused articles/GlobalPayoutGuide.md

```diff
@@ -90,21 +94,21 @@ Chimoney supports **Interledger Payment Pointers for Web Monetization**, enablin
 
 ## Getting Started with the Chimoney API
 
-Chimoney's Global Payouts API provides a seamless way for businesses to execute global payments effortlessly. This guide is tailored for developers, featuring step-by-step instructions on how to integrate Chimoney's API. Let's dive into the services you can harness the power of Chimoney's Global Payouts API to enhance your payment infrastructure. You will just need Chimoney account with API access , [Get Sandbox Access here](https://chimoney.readme.io/reference/sandbox-environment).
+Chimoney's Global Payouts API provides a seamless way for businesses to execute global payments effortlessly. This guide is tailored for developers, featuring step-by-step instructions on how to integrate Chimoney's API. Let's dive into the services you can harness the power of Chimoney's Global Payouts API to enhance your payment infrastructure. You will just need Chimoney account with API access.
```

Comment:
For this part, please change it to `To get started with the Chimoney API, follow this process:`